.releaserc

```diff
@@ -4,8 +4,8 @@
     "@semantic-release/release-notes-generator",
     ["@semantic-release/github", {
       "assets": [
-        {"path": "./roaster-*.xar", "label": "Expath package"}
-      ]  
-    }],
+        {"path": "./roaster-*.xar", "label": "Expath package (roaster-${nextRelease.gitTag}.xar)"}
```

Comment:
this will modify the string that appears in the Github UI, what you want (I think) is the `name:` property i.e. the file-name of the xar file in question.

Reply:
yes, I have not pushed that yet.

Reply:
then all is good, but i can't merge